app/views/components/_no_sam_header.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="usa-alert usa-alert-error auction-alert">
```

Comment:
bullet 1:  You have registered on SAM.gov 
bullet 2:  Your SAM.gov account is "active" in the system. Check your status using the SAM.gov status checker https://www.sam.gov/sam/helpPage/SAM_Reg_Status_Help_Page.html
bullet 3:  You have entered your DUNS number into your profile on this platform.